ports/atmel-samd/boards/trellis_m4_express/board.c

```diff
@@ -36,4 +41,11 @@ bool board_requests_safe_mode(void) {
 }
 
 void reset_board(void) {
+    uint8_t zeroes[96];
```

Comment:
`uint8_t zeroes[96] = {0};` works to init to zero also, and the compiler will choose the best way. Don't have to change it if you don't want to.